creator-node/src/services/stateMachineManager/processJob.ts

```diff
@@ -52,6 +53,7 @@ module.exports = async function (
     metricEndTimerFn({ uncaughtError: false })
     await redis.set(`latestJobSuccess_${queueName}`, Date.now())
   } catch (error: any) {
+    tracing.recordException(error)
```

Comment:
do we need to be concerned about internal errors inside this function?

Reply:
No, I'll look around for the doc that I read about this, but the opentelemetry library suppresses any internal exceptions so the worst that'll happen is a span won't get exported